shared/xccdf/system/software/integrity.xml

```diff
@@ -310,6 +310,45 @@ Bugzilla #1275532.
 <ref nist="AC-6,AU-9(1),AU-9(3),CM-6(d),CM-6(3)" disa="1494,1496" pcidss="Req-11.5" cis="1.2.6,6.1.3,6.1.4,6.1.5,6.1.6,6.1.7,6.1.8,6.1.9,6.2.3" srg="SRG-OS-000257-GPOS-00098,SRG-OS-000278-GPOS-00108" cjis="5.10.4.1" cui="3.3.8,3.4.1" />
 </Rule>
 
+<Rule id="rpm_verify_ownership" severity="high" prodtype="rhel7">
+<title>Verify and Correct Ownership with RPM</title>
+<description>
+The RPM package management system can check file ownership
+permissions of installed software packages, including many that are
+important to system security. After locating a file with incorrect
+permissions, which can be found with
+<pre>$ rpm -Va | grep "^.....\(U\|.G\)"</pre>
+
```

Comment:
Remove this whitespace

Reply:
Sorry which one?

Reply:
@bgjoseluis looks like there is whitespace on line 321.

Reply:
No, line 231 only have a line feed caracter.

Reply:
Then remove that. There should be no "empty" or "blank" lines inside `<description />`, `<rationale/>`, or `<ocil/>` elements.